service/controller/v2/arm_templates/main.json

```diff
@@ -402,7 +407,6 @@
           "vmssOsImageVersion": {"value": "[parameters('workerNodes')[0].osImage.version]"},
           "vmssOverprovision": {"value": "true"},
           "vmssVmCustomData": {"value": "[parameters('workerCloudConfigData')]"},
-          "vmssLbBackendPools": {"value": []},
```

Comment:
By default all load balancing managed by Kubernetes, so do not pass any backend configuration to workers